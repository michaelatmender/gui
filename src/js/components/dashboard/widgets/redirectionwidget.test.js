import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import { createMount } from '@material-ui/core/test-utils';
import RedirectionWidget from './redirectionwidget';

it('renders correctly', () => {
  const tree = createMount()(
    <MemoryRouter>
      <RedirectionWidget target="testlocation" buttonContent={<div />} />
    </MemoryRouter>
  );
  expect(tree.html()).toMatchSnapshot();
});
