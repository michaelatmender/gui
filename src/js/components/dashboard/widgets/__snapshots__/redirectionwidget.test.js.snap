// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `"<div class=\\"onboard widget\\"><div><p class=\\"muted\\"></p></div><a class=\\"MuiButtonBase-root MuiButton-root MuiButton-text\\" tabindex=\\"0\\" role=\\"button\\" aria-disabled=\\"false\\" href=\\"/testlocation\\"><span class=\\"MuiButton-label\\"><div></div></span><span class=\\"MuiTouchRipple-root\\"></span></a></div>"`;
